nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -134,6 +134,8 @@ In addition to numerous new and upgraded packages, this release has the followin
   [upstream's release notes](https://github.com/iputils/iputils/releases/tag/20221126)
   for more details and available replacements.
 
+- [services.xserver.videoDrivers](options.html#opt-services.xserver.videoDrivers) now defaults to the `modesetting` driver over device-specific ones. The `radeon`, `amdgpu` and `nouveau` drivers are still available, but effectively unmaintained and not recommended for use.
```

Comment:
> effectively unmaintained and not recommended for use

Is this your opinion, or is there upstream verbiage along those lines?

Reply:
I can't find direct statements from upstreams saying "don't use this", but upstreams have also all but disappeared.

Reply:
> I can't find direct statements from upstreams saying "don't use this", but upstreams have also all but disappeared.

`nouveau` just got updated in linux 6.2 and is expected to receive substantial updates in autumn/winter.

Reply:
This is the kernel bits, not the DDX.

Reply:
> This is the kernel bits, not the DDX.

I see, sorry.